wdl-lint/tests/lints/key-value-pairs/source.errors

```diff
@@ -0,0 +1,130 @@
+note[KeyValuePairs]: item should be followed by a newline
+  ┌─ tests/lints/key-value-pairs/source.wdl:8:22
+  │
+8 │         another_key: ["value1",
+  │                      ^
+  │
+  = fix: add a newline after this element
+
+note[KeyValuePairs]: incorrect indentation
+  ┌─ tests/lints/key-value-pairs/source.wdl:8:32
+  │  
+8 │           another_key: ["value1",
+  │ ╭───────────────────────────────^
+9 │ │         "value2", "value3"]
+  │ ╰────────^
+  │  
+  = fix: add 4 spaces to indentation
+
+note[KeyValuePairs]: item should be followed by a newline
+  ┌─ tests/lints/key-value-pairs/source.wdl:9:9
+  │
+9 │         "value2", "value3"]
+  │         ^^^^^^^^^^
+  │
+  = fix: add a newline after this element
+
+note[KeyValuePairs]: item should be followed by a newline
+  ┌─ tests/lints/key-value-pairs/source.wdl:9:19
+  │
+9 │         "value2", "value3"]
+  │                   ^^^^^^^^
+  │
+  = fix: add a newline after this element
+
+note[KeyValuePairs]: item should be followed by a newline
+   ┌─ tests/lints/key-value-pairs/source.wdl:10:19
+   │
+10 │         more_key: {d: "a",
+   │                   ^
+   │
+   = fix: add a newline after this element
+
+note[KeyValuePairs]: item should be followed by a newline
+   ┌─ tests/lints/key-value-pairs/source.wdl:11:13
+   │
+11 │             e: "b",}
+   │             ^^^^^^^
+   │
+   = fix: add a newline after this element
+
+note[KeyValuePairs]: item should be followed by a newline
+   ┌─ tests/lints/key-value-pairs/source.wdl:13:16
+   │
+13 │             a: {b: "c",
+   │                ^
+   │
+   = fix: add a newline after this element
+
+note[KeyValuePairs]: item should be followed by a newline
+   ┌─ tests/lints/key-value-pairs/source.wdl:14:17
+   │
+14 │                 d: "e",},
+   │                 ^^^^^^^
+   │
+   = fix: add a newline after this element
+
+note[KeyValuePairs]: incorrect indentation
+   ┌─ tests/lints/key-value-pairs/source.wdl:18:1
+   │
+18 │                 },
+   │ ^^^^^^^^^^^^^^^^
+   │
+   = fix: remove 4 spaces of indentation
+
+note[KeyValuePairs]: item should be followed by a newline
+   ┌─ tests/lints/key-value-pairs/source.wdl:19:16
+   │
+19 │             k: ["l",
+   │                ^
+   │
+   = fix: add a newline after this element
+
+note[KeyValuePairs]: item should be followed by a newline
+   ┌─ tests/lints/key-value-pairs/source.wdl:20:17
+   │
+20 │                 "m", "n"],
+   │                 ^^^^^
+   │
+   = fix: add a newline after this element
+
+note[KeyValuePairs]: item should be followed by a newline
+   ┌─ tests/lints/key-value-pairs/source.wdl:20:22
+   │
+20 │                 "m", "n"],
```

Comment:
It should also be followed by a comma. I just peeked lower and see you've `except`ed `TrailingComma`. Can you allow that rule?

Reply:
I was trying to suppress errors that weren't relevant to this rule. If I remove the exception, there are additional lints.
```
+note[TrailingComma]: item missing trailing comma
+  ┌─ tests/lints/key-value-pairs/source.wdl:9:19
+  │
+9 │         "value2", "value3"]
+  │                   ^^^^^^^^
+  │
+  = fix: add a comma after this element
```

and 

```
+note[TrailingComma]: item missing trailing comma
+   ┌─ tests/lints/key-value-pairs/source.wdl:20:22
+   │
+20 │                 "m", "n"],
+   │                      ^^^
+   │
+   = fix: add a comma after this element
+
```

and 

```
+note[TrailingComma]: item missing trailing comma
+   ┌─ tests/lints/key-value-pairs/source.wdl:34:13
+   │
+34 │             choices: ["yes", "reverse", "no"]
+   │             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   │
+   = fix: add a comma after this element
```